webpay/base/dev_messages.py

```diff
@@ -15,6 +15,7 @@
 BAD_REQUEST = 'BAD_REQUEST'
 BAD_SIM_RESULT = 'BAD_SIM_RESULT'
 BANGO_ERROR = 'BANGO_ERROR'
+BUYER_NOT_CONFIGURED = 'BUYER_NOT_CONFIGURED'
```

Comment:
Ok so, developer centric error messages go in this file. User centric error messages go in spartacus. I think.